deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -1437,6 +1437,12 @@ class ChatViewController: UITableViewController {
         } else {
             // new outgoing message
             let msg = dcContext.getMessage(id: messageId)
+            if msg.isInfo,
```

Comment:
I bet because we normally don't receive `DC_EVENT_MSGS_CHANGED` and `DC_EVENT_MSG_DELIVERED.` if a outgoing info message is sent. I think this is actually a core issue, cmp. https://github.com/deltachat/deltachat-ios/issues/1467#issuecomment-1034262342